microprofile/WFLY-20684_microprofile_platform_7.1.adoc

```diff
@@ -0,0 +1,147 @@
+---
+categories:
+- microprofile
+stability-level: default
+issue: https://github.com/wildfly/wildfly-proposals/issues/735
+feature-team:
+ developer: jasondlee
+ sme:
+  - kabir
+  - pferraro
+ outside-perspective:
+  -
+promotes:
+promoted-by:
+---
+= MicroProfile Platform 7.1
+:author:            Jason Lee
+:email:             jasondlee@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+
+== Overview
+
+This RFE is about updating WildFly to support MicroProfile Platform 7.1. We currently support MicroProfile Platform 7.0.
+
+The major.minor versions of the individual specs are shown below. Micro updates are strictly for TCK fixes, and we should use the latest for the TCKs as these versions become available.
+
+[cols="1,1,1,1,1"]
+|===
+|Spec | MP 7.0 Spec Version | Current Spec WildFly Version | MP 7.1 Spec Version | Updated Functionality (y/n)
+|Config|3.1|3.1|3.1|n
+|Fault Tolerance|4.1|4.1|4.1|n
+|Health|4.0|4.0|4.0|n
+|JWT|2.1|2.1|2.1|n
+|OpenAPI|4.0|4.0|4.1|y https://issues.redhat.com/browse/WFLY-20686[WFLY-20686]
+|Rest Client|4.0|4.0|4.0|n
+|Telemetry|2.0|2.0|2.1|y https://issues.redhat.com/browse/WFLY-20685[WFLY-20685]
+|===
+
+There is no additional work in this RFE beyond what is done by the individual spec update RFEs, i.e. it is just an umbrella issue.
+
+=== User Stories
+
+A user can use WildFly knowing that the specs making up the MicroProfile Platform have the versions in MicroProfile Platform 7.1.
+
+== Issue Metadata
+
+* https://issues.redhat.com/browse/WFLY-20684[WFLY-20684 - Implement MicroProfile Platform 7.1]
+
+=== Related Issues
+
+The main work is in these individual spec updates:
+
+* https://issues.redhat.com/browse/WFLY-20686[WFLY-20686]
+* https://issues.redhat.com/browse/WFLY-20685[WFLY-20685]
+
+=== Affected Projects or Components
+
+WildFly will be updated to pull in the corresponding versions of the Eclipse Microprofile specification jars, as well as the SmallRye jars and dependencies needed to implement each version.
+
+=== Other Interested Projects
+
+=== Relevant Installation Types
+
+
+* Traditional standalone server (unzipped or provisioned by Galleon)
+* OpenShift Source-to-Image (S2I)
+* Bootable jar
```

Comment:
I never remember if we should add domain mode here or not. I think ages ago we shouldn't but then I think recently @bstansberry might have mentioned that domain model should be listed anyway. Take this with a pinch of salt :-)